core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -361,8 +362,8 @@ object KafkaConfig {
   val ProcessRolesProp = "process.roles"
   val ControllerConnectProp = "controller.quorum.voters"
   val InitialBrokerRegistrationTimeoutMs = "initial.broker.registration.timeout.ms"
-  val RegistrationHeartbeatIntervalMsProp = "registration.heartbeat.interval.ms"
-  val RegistrationLeaseTimeoutMsProp = "registration.lease.timeout.ms"
+  val BrokerHeartbeatIntervalMsProp = "broker.heartbeat.interval.ms"
+  val BrokerRegistrationTimeoutMsProp = "broker.registration.timeout.ms"
```

Comment:
@cmccabe I think it's better to call this `broker.registration.lease.timeout.ms`. I left it to reflect to what is in the KIP today.

Reply:
the second one is now `broker.session.timeout.ms` since the KIP changed today :)